ref/api.md

```diff
@@ -506,6 +507,38 @@ Bugs and limitations:
     `hasTrailingZero` above, and
     [#178](https://github.com/scottlamb/moonfire-nvr/issues/178).
 
+### `DELETE /api/cameras/<uuid>/<stream>/view.mp4`
+
+Requires the `viewVideo` permission.
+
+Returns a `.mp4` file, with an etag and support for range requests. The MIME
+type will be `video/mp4`, with a `codecs` parameter as specified in
+[RFC 6381][rfc-6381].
+
+Expected query parameters:
+
+*   `s` (one or more): a string of the form `START_ID[-END_ID]`. This
+    specifies *recordings* to delete. The ids to retrieve are as returned by
+    the `/recordings` URL.
+
+Example request URI to delete recording id 1 from the given camera:
+
+```
+    /api/cameras/fd20f7a2-9d69-4cb3-94ed-d51a20c3edfe/main/view.mp4?s=1
+```
+
+Example request URI to delete recording ids 1–5 from the given camera
+
+```
+    /api/cameras/fd20f7a2-9d69-4cb3-94ed-d51a20c3edfe/main/view.mp4?s=1-5
+```
+
+Bugs and limitations:
+
+*   If the `s=` parameter references a recording id that doesn't exist when the
```

Comment:
I think it should just return success in this case: the range is empty on exit as desired. It'd also make the request idempotent (always nice) and means it won't fail due to a race with the normal rotation of oldest stuff.